### PR TITLE
SPEC-965 Publish server version, wire version, and feature table

### PR DESCRIPTION
**Pull request by rathisekaran**

### CODE REVIEW

File: source/wireversion-featurelist.rst [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+.. list-table::
+   :header-rows: 1
+
+   * - Server version
+     - Wire version
+     - Feature List
+
+   * - 2.4 and before
+     - 0
+     - Text search
```
Comment (behackett) [inline source/wireversion-featurelist.rst]: This is a bit misleading. The "text" command was added in 2.4, but doesn't exist in earlier versions. It's probably better not to list any features here.

File: source/wireversion-featurelist.rst [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+   * - 3.0
+     - 3
+     - | List collections
+       | List indexes
+       | SCRAM-SHA1
+       | explain command 
+
+   * - 3.2
+     - 4
+     - | (find/getMore/killCursors) commands
```
Comment (behackett) [inline source/wireversion-featurelist.rst]: currentOp and fsyncUnlock commands as well. Basically, anything that was previously implemented as a query on a special virtual collection became a command.

Comment (rathisekaran) [reply source/wireversion-featurelist.rst]: Done.

File: source/wireversion-featurelist.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+   * - 2.6 
+     - 2
+     - | Write commands (insert/update/delete)
+       | Aggregation $out pipeline operator
+  
+   * - 3.0
+     - 3
+     - | List collections
+       | List indexes
```
Comment (ShaneHarvey) [inline source/wireversion-featurelist.rst]: listCollections command, listIndexes command.

Comment (rathisekaran) [reply source/wireversion-featurelist.rst]: Updated and added listDatabases to 3.6 list

File: source/wireversion-featurelist.rst [hunk 4]
```diff
@@ -47,6 +47,7 @@ Server Wire version and Feature List
        | Retryable Writes
        | Causally Consistent Reads
        | Logical Sessions
+       | listDatabases
```
Comment (ShaneHarvey) [inline source/wireversion-featurelist.rst]: listDatabases existed before 2.4.

File: source/wireversion-featurelist.rst [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+   * - 2.6 
+     - 2
+     - | Write commands (insert/update/delete)
+       | Aggregation $out pipeline operator
+  
+   * - 3.0
+     - 3
+     - | listCollections
+       | listIndexes
+       | SCRAM-SHA1
```
Comment (behackett) [inline source/wireversion-featurelist.rst]: It's actually SCRAM-SHA-1.

Comment (rathisekaran) [reply source/wireversion-featurelist.rst]: Done. Merging PR

**Pull request merged**